packages/components/src/components/radio-button/radio-button.stories.ts

```diff
@@ -0,0 +1,190 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-radio-button');
+const { overrideArgs } = storybookHelpers('sd-radio-button');
+const { generateTemplate } = storybookTemplate('sd-radio-button');
+
+export default {
+  title: 'Components/sd-radio-button',
+  component: 'sd-radio-button',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'icon',
+      value: '<sd-icon library="global-resources" name="system/picture" slot="icon"></sd-icon>'
+    },
+    {
+      type: 'slot',
+      name: 'default',
+      value: '<div>Label</div>'
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+
+/**
+ * Default: This shows sd-radio-button in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * Use the `size` attribute to change the size of the input radio. This attribute affects the font-size within the element, while the element itself remains the same size.
+ */
+
+export const Size = {
+  parameters: { controls: { exclude: ['size'] } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        x: { type: 'attribute', name: 'size' }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * Use the disabled attribute to disable a input radio. Clicks will be suppressed until the disabled state is removed
+ */
+
+export const DisabledAndSize = {
+  name: 'Disabled × Size',
+  parameters: { controls: { exclude: ['disabled', 'size', 'default'] } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'disabled',
+          values: [false, true]
+        },
+        y: {
+          type: 'attribute',
+          name: 'size',
+          values: ['lg', 'md', 'sm']
+        }
+      },
+      constants: [{ type: 'attribute', name: 'disabled', value: true }],
+      args
+    });
+  }
+};
+
+/**
+ * Use the show-label attribute to show the label of the radio button.
+ */
+
+export const CombinationAndSize = {
+  name: 'Combination × Size',
```

Comment:
As this story describes the `showLabel` attribute, I would like to see the story naming describing it more clear.

Reply:
```suggestion
  name: 'Label × Size',
```

Reply:
or something like 'Additional Label x Size'